pyrometheus/__init__.py

```diff
@@ -63,52 +67,84 @@ def str_np_inner(ary):
 def str_np(ary):
     return "np.array(%s)" % str_np_inner(ary)
 
+
 # }}}
 
 
 # {{{ polynomial processing
 
+
 def nasa7_conditional(t, poly, part_gen):
     # FIXME: Should check minTemp, maxTemp
     return p.If(
-            p.Comparison(t, "<", poly.coeffs[0]),
-            part_gen(poly.coeffs[1:8], t),
-            part_gen(poly.coeffs[8:15], t))
+        p.Comparison(t, ">", poly.coeffs[0]),
+        part_gen(poly.coeffs[1:8], t),
+        part_gen(poly.coeffs[8:15], t),
+    )
 
 
 @singledispatch
 def poly_to_expr(poly):
-    raise TypeError(f"unexpected argument type in poly_to_expr: {type(poly)}")
+    message = f"unexpected argument type in poly_to_expr: {type(poly)}"
+    raise TypeError(message)
```

Comment:
Could you explain this change?

Reply:
The `raise TypeError(f"...")` line was too long for all the `poly_to_*_expr` functions, so I ended up breaking it into two lines, as you see here. However, line length was not a problem with this one, so this change is for consistency.

Reply:
9770488